### PR TITLE
This is just a start

### PR DESCRIPTION
**Pull request by FDiskas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,99 @@
-# Exoframe Website
+### Exoframe Website
 
+<img alt="Exoframe" src="https://github.com/exoframejs/exoframe/raw/master/logo/png/exo_blue.png" width="300">
+
+> Simple Docker deployment tool
+
+[![Build Status](https://travis-ci.org/exoframejs/exoframe.svg?branch=master)](https://travis-ci.org/exoframejs/exoframe)
```
Comment (yamalight) [inline README.md]: Those probably can be removed since they show status of CLI 🤔

File: README.md [hunk 2]
```diff
@@ -1,2 +1,99 @@
...
+[![Coverage Status](https://coveralls.io/repos/github/exoframejs/exoframe/badge.svg?branch=master)](https://coveralls.io/github/exoframejs/exoframe?branch=master)
+[![npm](https://img.shields.io/npm/v/exoframe.svg)](https://www.npmjs.com/package/exoframe)
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg?maxAge=2592000)](https://opensource.org/licenses/MIT)
+
+Exoframe is a self-hosted tool that allows simple one-command deployments using Docker.
```
Comment (yamalight) [inline README.md]: This (and pretty much everything else in this readme) should be part of index page of the website

File: src/models/AllMarkdownRemark.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import Post from './Post';
+
+interface AllMarkdownRemark {
```
Comment (yamalight) [inline src/models/AllMarkdownRemark.ts:3]: I assume those are needed for typescript, right?

File: src/pages/contents/Advanced.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
+---
```
Comment (yamalight) [inline src/pages/contents/Advanced.md]: Can we have content under `./content` instead of `./src/pages/content`, please?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  "author": "",
+  "keywords": [
+    "exoframe"
+  ],
+  "license": "MIT",
+  "main": "n/a",
+  "scripts": {
+    "build": "gatsby build",
+    "start": "gatsby develop",
+    "fix": "prettier --write 'src/**/*.{ts,tsx,scss,md}' && tslint --fix 'src/**/*.{ts,tsx}' -p tsconfig.json && stylelint --fix 'src/**/*.scss' "
```
Comment (yamalight) [inline package.json]: Please also add "lint" command that runs tslint / eslint

**Review by yamalight — CHANGES_REQUESTED** [review]
Awesome work!
Got a few minor comments.

One more awesome thing to have (in case you have a bit more time 😉) would be basic github actions setup (just "build" for everything to make sure it doesn't break, and some sort of release pipeline for github pages from tags)

**Comment by FDiskas** [comment]
Sorry - but github pages you should do by your self. It requires change repo settings, grab api key and pas it to action. :hourglass_flowing_sand:

**Review by yamalight — APPROVED** [review]
Looks great, thanks!

**Pull request merged**